docs/api/platform/PowerManagement.md

```diff
@@ -81,6 +81,22 @@ UNLOCK: mbed_rtx_idle.cpp, ln: 131, lock count: 1
 
 <span class="notes">**Note:** Sleep tracing is a debug feature and should only be enabled during development cycle. Its heavy use of UART can affect the device performance.</span>
 
+There is a small trade-off between power saving and memory footprint. When tickless mode is enabled, the memory footprint of your application is slightly bigger. Therefore if tickless mode does not fit your application requirement, for example you may rely on SysTick always being present, then you can disable tickless and achieve further memory optimisation. To do that, you can override the default configuration in your application `mbed_app.json` like this:
+
+```json
+{
+    "macros_remove": ["MBED_TICKLESS"]
+}
+```
+
+https://github.com/ARMmbed/mbed-os-example-blinky application compiled with the GCC_ARM toolchain for some targets.
```

Comment:
I would remove these information as it is bound to go out of date. I'd rather quote the range of difference eg. between 500B and 1k

Reply:
[Done](https://github.com/ARMmbed/mbed-os-5-docs/compare/2015e28ce4d2149a6760b34709b4fb4386329b52..e46ca9f89262cd51257d899408d1442c694c0b6a).